sdk/js-cloud-server/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { DevCycleOptions } from './types'
+import { DevCycleCloudClient } from './cloudClient'
+import { isValidServerSDKKey } from './utils/paramUtils'
+import { DevCycleUser } from './models/user'
+
+export { DevCycleCloudClient, DevCycleUser }
+export * from './models/populatedUser'
+export * from './models/user'
+export * from './models/variable'
+export * from './types'
+export * from './request'
+export * from './utils/logger'
+export * from './utils/paramUtils'
+
+type DevCycleCloudOptions = DevCycleOptions & {
+    platform?: 'NodeJS' | 'Electron' | 'EdgeWorker'
```

Comment:
are we adding these platforms to the dashboard platform filter options?

Reply:
We can yea